app-text/aspell/aspell-0.60.6.1.recipe

```diff
@@ -12,9 +12,10 @@ more than one Aspell process is open at once."
 HOMEPAGE="http://aspell.net"
 COPYRIGHT="2000-2006 Kevin Atkinson"
 LICENSE="GNU LGPL v2"
-REVISION="3"
-SOURCE_URI="ftp://ftp.gnu.org/gnu/aspell/aspell-0.60.6.1.tar.gz"
+REVISION="4"
+SOURCE_URI="ftp://ftp.gnu.org/gnu/aspell/aspell-$portVersion.tar.gz"
```

Comment:
Can we switch to github?
https://github.com/GNUAspell/aspell/archive/rel-0.60.6.1.tar.gz

Reply:
For what? 
Official page is here: http://aspell.net/
The link on official page leads to gnu.org.
So, i do not see any reason this is the case to NOT merge this pr.

Reply:
sry, found link on github on official page :)

Reply:
Ok, we CAN. But github page  has no packet signatures. The tarballs are generated from git tree (binded to tags). So, those tarballs not the same on the gnu ftp...
So, in my opinion, we should stay on ftp.gnu.org...

Anyway, technically, we can do that. So, what shall i do?